README.md

```diff
@@ -132,6 +132,10 @@ Please take a quick look at the [contribution guidelines](/CONTRIBUTING.md) firs
 
 * [Freezed](https://github.com/rrousselGit/freezed) - Code generation for immutable classes that has a simple syntax/API without compromising on the features.
 
+## Crash monitoring
+
+* [Sentry](https://sentry.io/) - Sentry provides self-hosted and cloud-based error monitoring that helps all software teams discover, triage, and prioritize errors in real-time.
```

Comment:
Thanks for the contribution! Can you link to https://github.com/getsentry/sentry-dart instead please?

Reply:
thanks and done :)